controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook.go

```diff
@@ -74,6 +74,9 @@ const (
 	spec                 = "spec"
 	kubeadmConfigSpec    = "kubeadmConfigSpec"
 	clusterConfiguration = "clusterConfiguration"
+	preKubeadmCommands   = "preKubeadmCommands"
+	postKubeadmCommands  = "postKubeadmCommands"
+	files                = "files"
```

Comment:
The above 3 field names are used only in one place, so I don't think it is required a constant (and for consistency, there are no constant for many other fields with the exception of top level entries)
So I IMO this should be removed

Reply:
@fabriziopandini thanks for review, and to add, these 3 field names are in the same level as `clusterConfiguration`, which has a constant as well as for consistency.

Reply:
Ok, this makes sense in its way